src/vendor/ZSDK.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable */
+/* eslint-disable */ // remove this when building
```

Comment:
Did you have trouble with this as well?

Reply:
I'm not sure, I never built because I used the local. This was based off your advice